tests/gui/test_sensors_panel.py

```diff
@@ -0,0 +1,80 @@
+"""Tests for the SensorsPanel."""
+
+from unittest.mock import Mock, call, patch
+
+from PySide6.QtWidgets import QLabel, QLineEdit
+
+from finesse.config import SENSORS_TOPIC
+from finesse.gui.sensors_panel import SensorsPanel
+from finesse.sensor_reading import SensorReading
+
+
+@patch("finesse.gui.sensors_panel.pub.subscribe")
+def test_init(subscribe_mock, qtbot) -> None:
+    """Test SensorsPanel's constructor."""
+    panel = SensorsPanel()
+    assert panel.title() == "Sensor readings"
+
+    # Check the panel's widgets are created correctly
```

Comment:
I wondered if it was worth making any assertions about the other widgets in the panel (`_reading_layout` etc), but given that they're all basically empty I wasn't sure how much value it would add.

Reply:
I agree. It would be a pain to check everything, so I think it's fine to just check the main moving parts (e.g. labels that change etc.).